src/main/java/com/scad/expensestracker/controllers/ExpensesTrackerItemController.java

```diff
@@ -21,82 +21,71 @@
 import com.scad.expensestracker.models.ExpensesTrackerItem;
 import com.scad.expensestracker.repositories.ExpensesTrackerRepository;
 
-import ch.qos.logback.core.model.Model;
 import jakarta.validation.Valid;
 
 @Controller
 public class ExpensesTrackerItemController {
     private final Logger logger = LoggerFactory.getLogger(ExpensesTrackerItemController.class);
-   
-    @Autowired // This allows Spring to manage the lifecycle of the repository and ensures that 
-    // it's properly initialized when it's used in your controller.
-    private ExpensesTrackerRepository ExpensesTrackerRepository;
+
+    @Autowired
+    private ExpensesTrackerRepository expensesTrackerRepository;
 
     @GetMapping("/")
-    public ModelAndView index(Model model){
+    public ModelAndView index() {
         logger.debug("root to GET index");
         ModelAndView modelAndView = new ModelAndView("index");
-        List<ExpensesTrackerItem> expensesTrackerItems = new ArrayList<>();
-        ExpensesTrackerRepository.findAll().forEach(expensesTrackerItems::add);
+        List<ExpensesTrackerItem> expensesTrackerItems = (List<ExpensesTrackerItem>) expensesTrackerRepository.findAll();
```

Comment:
What the advantages of using list over array?

Reply:
that array previously was empty , i initialized list items by fetching all items n expenses list